build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+pip install -r requirements.txt
```

Comment:
For the same reasons as above, this line can probably be removed.

Reply:
Fixed it